avalanche/training/templates/observation_type/batch_observation.py

```diff
@@ -12,8 +6,8 @@
 
 
 class BatchObservation(SGDStrategyProtocol):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
if we are using only kwargs shouldn't we have only kwargs in the method signature and super call?
btw, maybe add a small comment to explain this in the template classes.